playground/src/utils/helper_functions.tsx

```diff
@@ -21,121 +22,175 @@ export interface checkedData {
+  if (inputString === '') {
+    return { isValid: true, errorMessage: '' };
   }
 
-  if (
-    Object.values(jsonInput).length === 0 ||
-    Object.values(jsonInput).some(isNaN)
-  ) {
-    const errorMessage = `${key} must contain at least one number,
-and it can only contain numbers.`;
-
-    return { isValid: false, errorMessage: errorMessage };
-  }
-
-  return { isValid: true, errorMessage: "" };
-}
-
-/**
- * We check the inputs and return true or false. We allow:
- * - an empty input, and
- * - a valid JSON object containing stack_init or advice_tape (or both)
- * if the values are numbers
- */
-export function checkInputs(jsonString: string): checkedData {
-  if (jsonString === "") {
-    return { isValid: true, errorMessage: "" };
-  }
-
-  let jsonInput!: JSON;
-
+  // Parse the input string to JSON
+  let input: JSON;
   try {
-    jsonInput = JSON.parse(jsonString);
-  } catch (e: any) {// eslint-disable-line @typescript-eslint/no-explicit-any
-
-    const errorMessage = `Miden VM Inputs need to be a valid JSON object:
-${e.message}`;
+    input = JSON.parse(inputString);
+  } catch (error: any) {
+    const errorMessage = `Inputs must be a valid JSON object: ${error.message}`;
     return { isValid: false, errorMessage: errorMessage };
   }
 
-  if (
-    !Object.keys(jsonInput).includes("stack_init") &&
-    !Object.keys(jsonInput).includes("advice_tape")
-  ) {
-    const errorMessage = `Miden VM Inputs can be empty or
-we need either a stack_init or
-an advice_tape.`;
-
-    return { isValid: false, errorMessage: errorMessage };
+  try {
+    inputSchema.validateSync(input, { strict: true, stripUnknown: false });
+    return { isValid: true, errorMessage: '' };
+  } catch (error: any) {
+    return { isValid: false, errorMessage: `Invalid inputs: ${error.message}` };
   }
+};
 
-  return checkFields(jsonInput);
-}
+const outputSchema = yup.object().shape({
+  stack_output: yup.array().of(yup.number().integer().min(0)).required(),
+  overflow_addrs: yup.array().of(yup.number().integer().min(0)).required(),
+  trace_len: yup.number().integer().min(0).optional(),
+});
 
 /**
  * We check the outputs and return true or false. We allow:
  * - a valid JSON object containing stack_output (and overflows if present)
  * - only numbers as values
  */
 export function checkOutputs(jsonString: string): checkedData {
-  if (jsonString === "") {
-    const errorMessage = `We need some outputs to verify the program.
-Did you prove the program first?`;
-
+  if (jsonString === '') {
+    const errorMessage = `We need some outputs to verify the program. Did you prove the program first?`;
     return { isValid: false, errorMessage: errorMessage };
   }
 
+  // Parse the json output
   let jsonOutput!: JSON;
-
   try {
     jsonOutput = JSON.parse(jsonString);
-  } catch (e: any) {// eslint-disable-line @typescript-eslint/no-explicit-any
+  } catch (e: any) {
+    // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
does this still work to disable linting for the line with the `any` type? I thought it had to be on the same line (or on the line before it using `eslint-disable-next-line` instead of `eslint-disable-line`)

It would be better to just use an `Error` type instead of `any`, but that doesn't have to happen in this Pr